docker/neutron/neutron-mlnx-agent/Dockerfile.j2

```diff
@@ -0,0 +1,46 @@
+FROM {{ namespace }}/{{ image_prefix }}neutron-base:{{ tag }}
+LABEL maintainer="{{ maintainer }}" name="{{ image_name }}" build-date="{{ build_date }}"
+
+{% block neutron_mlnx_agent_header %}{% endblock %}
+
+{% import "macros.j2" as macros with context %}
+
+{% if base_package_type == 'rpm' %}
+
+    {% set neutron_mlnx_agent_packages = [
+        'libvirt-python',
+        'python-ethtool',
+    ] %}
+
+{% elif base_package_type == 'deb' %}
+
+    {% set neutron_mlnx_agent_packages = [
+        'python-libvirt',
+        'python-ethtool',
+    ] %}
+
+{% endif %}
+
+{% if install_type == 'binary' %}
+
+    {% set neutron_mlnx_agent_packages = neutron_mlnx_agent_packages + [
+        'python-networking-mlnx'
```

Comment:
thinking about it, do we need to do a version check - this should be python2-networking-mlnx on centos 7, right?

Reply:
you're right!